### ISSUE TITLE
Standalone executable crashes when gazebo is not yet started

### ISSUE THREAD

**Issue by osrf-migration** [osrf/gazebo#573]
**Original report ([archived issue](https://osrf-migration.github.io/gazebo-gh-pages/#!/osrf/gazebo/issues/573)) by Dereck Wonnacott (Bitbucket: [dawonn](https://bitbucket.org/%7Ba9f76668-8beb-4cc2-a518-99a40d29a8fc%7D/)).**

----------------------------------------
If you try to ::init() before gazebo has started, it promptly dies.  Here's a workaround but I'm not sure exactly how to 'fix' it...

```
#!diff

hg diff
diff -r c38c361cd0f5aa5acac19cc5aad91643f105d2b1 gazebo/transport/Connection.cc
--- a/gazebo/transport/Connection.cc	Sat Mar 09 15:02:07 2013 -0500
+++ b/gazebo/transport/Connection.cc	Sat Mar 09 20:48:47 2013 -0500
@@ -148,8 +148,8 @@
   if (!this->connectCondition.timed_wait(lock,
         boost::posix_time::milliseconds(60000)) || this->connectError)
   {
-    gzlog << "Failed to create connection to remote host["
-          << host << ":" << _port << "]\n";
+    //gzlog << "Failed to create connection to remote host["
+    //      << host << ":" << _port << "]\n";
     this->socket->close();
     return false;
   }
```

Basically, gzlog has not yet been initialized when the initial connection to Master is attempted, so it bombs.
